fastcrypto/src/groups/multiplier/integer_utils.rs

```diff
@@ -93,7 +93,9 @@ pub fn get_bits_from_bytes(bytes: &[u8], start: usize, end: usize) -> usize {
 /// Return true iff the bit at the given index is set.
 #[inline]
 pub fn test_bit(bytes: &[u8], index: usize) -> bool {
-    assert!(index < 8 * bytes.len());
+    if index >= 8 * bytes.len() {
```

Comment:
it's a bit unexpected, either:
- change the return value to Error
- clarify in the documentation (and function name?) that it treats the input as infinitely padded with zeros